src/vips/Image_methods.lua

```diff
@@ -158,10 +158,8 @@ function Image.new_from_buffer(data, options, ...)
     return voperation.call(name, options or "", data, unpack { ... })
 end
 
-function Image.new_from_memory(data, width, height, bands, format)
+function Image.new_from_memory(data, size, width, height, bands, format)
```

Comment:
This will change the API, of course. Could you add a new method with the extra parameter instead? Perhaps `new_from_memory_size`? The old `new_from_memory` method could just call that with `ffi.sizeof`.